### DISCUSSION TITLE
insert_buffer

### DISCUSSION THREAD

**Discussion by AayushiRV** [The-OpenROAD-Project/OpenROAD#3732] [General]
I am inserting buffers to my design. How can I find that buffers are being inserted? I cannot find anything in my design's netlist.

**Comment by maliberty** [comment]
How are you inserting the buffers?

**Reply by AayushiRV** [reply]
I am using "insert_buffer" command. 
![Screenshot from 2023-0](https://github.com/The-OpenROAD-Project/OpenROAD/assets/134275400/a951b750-0589-4a8a-afb3-fb7f90fc3ce2)

Am I doing anything wrong?

**Reply by vijayank88** [reply]
Where you got reference for above format?

**Reply by AayushiRV** [reply]
I got confused, it must be there in older version.

**Comment by maliberty** [comment]
There is no such command in OR:
```
openroad> insert_buffer
invalid command name "insert_buffer"
while evaluating insert_buffer
```

**Reply by AayushiRV** [reply]
Is there any other option to insert buffer in OR?? I am getting the max cap, max slew and max fanout in my design. I want to reduce the errors using ECO. Kindly help.

**Reply by vijayank88** [reply]
Which flow you're using `OpenROAD-flow-scripts` or `OpenLane`?

**Reply by AayushiRV** [reply]
OpenLane

**Reply by vijayank88** [reply]
Are you using latest OpenLane?

**Reply by AayushiRV** [reply]
Placement and routing for my design is not working in the latest version of OpenLane, I had already discussed it here-
https://github.com/The-OpenROAD-Project/OpenLane/issues/1911#issuecomment-1649153699 
So I am working with the older version.

**Reply by vijayank88** [reply]
Added my comment there. Try to resolve and then run the flow. 
As per https://github.com/The-OpenROAD-Project/OpenLane/pull/1685 PR ECO flow removed from OpenLane, so what ever you're trying its lead to an bug.

**Comment by vijayank88** [comment]
@AayushiRV 
`insert_buffer` is maintained by OpenLane. Check this example: https://github.com/The-OpenROAD-Project/OpenLane/blob/master/tests/1007/interactive.tcl
So OpenROAD nothing to do with that.
